src/CaptureClient/CompositeEventProcessor.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "CaptureClient/CaptureEventProcessor.h"
+
+namespace orbit_capture_client {
+
+namespace {
+
+class CompositeCaptureEventProcessor : public CaptureEventProcessor {
```

Comment:
The name of the file doesn't reflect the name of the class (or vice versa).

Reply:
done.